tfjs-layers/src/layers/preprocessing/image_resizing.ts

```diff
@@ -99,7 +91,7 @@ export class Resizing extends Layer {
         return image.resizeNearestNeighbor(
             inputs, size, !this.cropToAspectRatio);
       } else {
-        throw new Error('interpolation error encountered in image_resizing.ts');
+        throw new Error(`mode is ${this.interpolation} but only ${[...INTERPOLATION_METHODS]} are supported`);
```

Comment:
Nit: Sorry, my last suggestion forgot to include interpolation.
```suggestion
        throw new Error(`Interpolation is ${this.interpolation} but only ${[...INTERPOLATION_METHODS]} are supported`);
```